src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,330 @@
+* Their PayID
+* Identifying themselves with the wallet they end up redirected to in order to approve transfer of funds to the receiver.
+
+### No server-side software specific to PayID or its protocols for servicing the transaction
+
+Since the flow of PayID Easy Checkout is predicated on using the PayID Discovery Protocol and then redirecting the 
+sender away from the site of the receiver, all of the flow can be instrumented in the browser and doesn't require server-side resources. 
+
+Apart from a PayID Discovery compliant PayID Server, The PayID Easy Checkout Protocol does not require server-side 
+software to be run by either the sender or receiver for a transaction. The PayID server is capable of providing details 
+of where to send the user via the PayID Discovery Protocol. Assuming the wallet used by the sender has implemented 
+support in their UI for the PayID Easy Checkout Protocol, the sender can be redirected to their wallet within the 
+browser to complete their transaction.
+
+# Example Usage
+This section shows the canonical PayID Easy Checkout flow between a hypothetical merchant and customer. The merchant
+accepts payments at the PayID pay$merchant.com, and the customer has the PayID alice$wallet.com.
+
+## PayID Easy Checkout Initiation
+In this example, the customer might place some items in their online cart on the merchant's website, then choose
+to checkout.  The merchant would then render a form asking for the customer's PayID, as well as a "Checkout with PayID"
+button.  Once the user inputs their PayID alice$wallet.com and clicked the "Checkout with PayID" button, the merchant
+site begins the PayID Easy Checkout flow.
+
+## PayID Easy Checkout Wallet Discovery
+The merchant site would resolve the customer's PayID to a host as defined in [PAYID-URI][], in this case resolving
+alice$wallet.com to https://wallet.com. The merchant site would then perform PayID Discovery as defined in
+[PAYID-DISCOVERY][] to receive a PayID Easy Checkout JRD like this:
+    
+    GET /.well-known/webfinger?resource=payid%3Aalice%24wallet.com
+    Host: wallet.com
+    
+If the server has enabled PayID Easy Checkout in their wallet, they would respond with something like this:
+     
+     HTTP/1.1 200 OK
+     Access-Control-Allow-Origin: *
+     Content-Type: application/jrd+json
+
+     {
+       "subject" : "payid:alice$wallet.com",
+       "links" :
+       [
+         {  
+           "rel": "https://payid.org/ns/payid-easy-checkout/1.0",
+           "template": "https://wallet.com/checkout?amount={amount}&receiverPayId={receiverPayId}&currency={currency}&nextUrl={nextUrl}"
+         }
+       ]
+     }
+
+## Expand Wallet Discovery URL Template
+The merchant would parse the PayID Discovery response and iterate over the "links" collection to find the link with 
+the Relation Type of "https://payid.org/ns/payid-easy-checkout/1.0". The site can then do a search and replace on
```

Comment:
We should be consistent in usage with the word "site" (probably should be just "merchant"), esp if we want to leave room for an app doing this?

Reply:
what about defining "merchant" as the actual entity, like Amazon, and defining "merchant front end" to be the merchant's site/app etc, like amazon.com?

Reply:
calling this the UI, eg "merchant UI" and "wallet UI"